packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -419,13 +411,15 @@ export async function transformationJobErrorHandler(error: any) {
                 }
             })
     } else {
+        transformByQState.setToCancelled()
+        transformByQState.setPolledJobStatus('CANCELLED')
```

Comment:
Do we need both?

Reply:
Yes, in the job history panel we display the `polledJobStatus` so need this set too for the text/display purposes.